services/py-api/src/database/repository/participants_repository.py

```diff
@@ -48,12 +46,29 @@ async def create(
                 # If the team_id is passed as a kwarg the participant will be inserted in the given team
                 team_id=kwargs.get("team_id"),
             )
-            LOG.debug("Inserting participant...", participant=participant.dump_as_json())
+            if session:
+                LOG.debug(
+                    "Inserting participant {participant} via transaction {id}",
+                    participant=participant.dump_as_json(),
+                    id=session.transaction_id,
+                )
+
             await self._collection.insert_one(document=participant.dump_as_mongo_db_document(), session=session)
             return Ok(participant)
         except DuplicateKeyError:
-            LOG.warning("Participant insertion failed due to duplicate email", email=input_data.email)
+            if session:
+                LOG.warning(
+                    "Participant insertion failed due to duplicate email {email} via transaction {id}",
+                    email=input_data.email,
+                    transaction_id=session.transaction_id,
+                )
+
             return Err(DuplicateEmailError(input_data.email))
         except Exception as e:
-            LOG.exception("Participant insertion failed due to err {}".format(e))
+            if session:
+                LOG.exception(
+                    "Participant insertion failed due to err {e} via transaction {id}",
```

Comment:
You do not need the things in the {}, they are not replaced with the values, they just sit there as strings